README.md

```diff
@@ -3,53 +3,57 @@
 --->
 <!---Short description of what the extension allows the user to do and key features in bullet points below 
 --->
-The IBM Blockchain extension allows a user to write and test chaincode projects in Visual Studio Code:
-* Generate skeleton chaincode projects
+The IBM Blockchain extension allows a user to easily write and test smart contract projects in Visual Studio Code:
+* Generate skeleton smart contract projects
 * Adds a Fabric view to connect and interact with Fabric Blockchain networks
```

Comment:
When you mouse-over it, you'll see "IBM Blockchain Platform", so probably best to re-name it here as "Adds an IBM Blockchain Platform view".  And then we should say "Hyperledger Fabric" the first time we refer to it on a docs page.